bin/cli.js

```diff
@@ -2,39 +2,84 @@
 
 const { Command } = require('commander');
 const { spawn } = require('child_process');
+const debug = require('debug')('laravel-mix');
 
-const program = createProgram();
+run();
 
-executeScript();
+async function run() {
+    // TODO: This really shouldn't be here I don't think
+    require('../src/index');
+
+    const mixConfigPath = Mix.paths.mix();
+    const webpackConfigPath = require.resolve('../setup/webpack.config.js');
 
-function createProgram() {
     const program = new Command();
 
     program.name('mix');
     program.version('0.0.1');
+    program.option(
+        '--mix-config <path>',
+        'Specify mix config file',
+        mixConfigPath
+    );
+    program.option(
+        '--webpack-config <path>',
+        'Specify webpack config file',
+        webpackConfigPath
```

Comment:
So what's the usecase for allowing a user to override the webpack config file that Mix uses?

Reply:
Thinking about it… it shouldn't ever need to change. Not even for testing so I've removed this option.